src/main/java/de/tum/cit/aet/artemis/core/config/Constants.java

```diff
@@ -374,6 +374,11 @@ public final class Constants {
      */
     public static final String ACTIVE_MODULE_FEATURES = "activeModuleFeatures";
 
+    /**
+     * The name of the module feature used for Passkey functionality.
+     */
+    public static final String FEATURE_ATLAS = "passkey";
```

Comment:
I think this is a typo, or is ATLAS == PASSKEY

Reply:
thanks :-)